### ISSUE TITLE
Bug sur show_balloon

### ISSUE THREAD

**Issue by Zer0zer0x** [RMEx/RME#230]
Hello !

```
Déplacer un événement (cet événement)
 show_balloon(me, 3, true)
```

Retourne l'erreur : 
Commands.rb line 1720: FiberError occured.
Can't yield from root fiber.

```
Déplacer un événement (cet événement)
 show_balloon(me, 3)
```

Fonctionne correctement...
J'en infère que le wait_flag buggue quand il est appelé par un déplacement d'événement ?

Bon courage ! o/

**Comment by BastienDuplessier** [comment]
Oui, on ne peut pas utiliser le wait_flag dans un Déplacer évènement. Malheureusement on ne peut pas y faire grand chose :/
Utilise plutôt l'option "Attendre la fin" dans la commande Déplacer évènement.

**Comment by Zer0zer0x** [comment]
Oki. :3 
C'est déjà fait, pour ma part. ^^

Bon courage, si jamais une idée vous vient ! :)
Ou alors... peut-être le préciser dans la doc ? :)

**Comment by BastienDuplessier** [comment]
Je pense qu'on va l'indiquer dans la doc oui.
